modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/js/message_boards/MBPortlet.js

```diff
@@ -246,42 +245,6 @@ class MBPortlet {
 		}
 	}
 
-	/**
-	 * Updates the attachments to include the checked attachments.
-	 *
-	 */
-
-	_updateMultipleMBMessageAttachments() {
-		const namespace = this._namespace;
-
-		const selectedFileNameContainer = document.getElementById(
-			`${namespace}selectedFileNameContainer`
-		);
-
-		if (selectedFileNameContainer) {
-			const inputName = `${namespace}selectUploadedFile`;
-
-			const input = [].slice.call(
-				this.rootNode.querySelectorAll(
-					`input[name=${inputName}]:checked`
-				)
-			);
-
-			const data = input
-				.map((item, index) => {
-					const id = index;
-					const value = item.value;
-
-					return `<input id="${namespace}selectedFileName${id}" name="${namespace}selectedFileName" type="hidden" value="${escapeHTML(
```

Comment:
Isn't the original issue here? The `escapeHTML` call will not escape commas; if instead of doing this you escape the text following the rules for URI components it should work as expected.

Reply:
Hey @adolfopa ,

I think the problem is that I didn't explain the issue in a correct way in the description of this PR, sorry for that. The original issues was that when we were using the innerHTML function could cause an XSS attack issue because as we were adding the filename without escaping users could add an script as a file name, so for that reason we included the escapeHTML function in order to do not allow something like that. Now, like we don't need to include input hidden attributes in the dom, this issues no longer exists.

Let me know If I didn't explain correctly and furthermore I think that we have other scenarios or uses cases that we have to fix this issue in a correct way.

Regards!